frontend/src/components/Start.js

```diff
@@ -0,0 +1,229 @@
+import styled from 'styled-components/macro'
+import startscreenJpg from './../assets/startscreen.jpg'
+import wandergoldSvg from './../assets/wandergold.svg'
+import closeSvg from './../assets/close.svg'
+import compassSvg from './../assets/compass.svg'
+import { useState, useEffect } from 'react'
+
+export default function Start() {
+  const [isSearchFocused, setSearchFocus] = useState(false)
+  const [coordsToSearch, setCoordsToSearch] = useState({
+    latitude: null,
+    longitude: null,
+    locationName: '',
+    googlePlaceId: '',
+  })
+  const [suggestionList, setSuggestionList] = useState([])
+
+  useEffect(() => console.log(coordsToSearch))
+
+  return (
+    <Wrapper>
+      <LogoArea onClick={() => setSearchFocus(false)}>
+        <h1>
+          <img src={wandergoldSvg} alt="wandergold" />
+        </h1>
+      </LogoArea>
+      <LocationSearch>
+        <SearchField className={isSearchFocused && 'active'}>
+          {isSearchFocused && (
+            <button type="button" onClick={() => setSearchFocus(false)}>
+              <img src={closeSvg} alt="close" />
+            </button>
+          )}
+          <input
+            onChange={(event) => getSuggestions(event.target.value)}
+            onFocus={() => setSearchFocus(true)}
+            type="text"
+            placeholder={!isSearchFocused && 'wo willst du hin?'}
+          ></input>
+        </SearchField>
+        {isSearchFocused && (
+          <SearchSuggestions>
+            <ul>
+              <>
+                {navigator.geolocation && (
+                  <li
+                    className="geoLocator"
+                    onClick={() => getGeolocationOfUser()}
+                  >
+                    Mein Standort
+                  </li>
+                )}
+
+                {suggestionList.map(({ description, place_id }) => (
+                  <li
+                    onClick={() =>
+                      setCoordsToSearch({
+                        latitude: 51.111,
+                        longitude: 12.22323,
+                        description: description,
+                        googlePlaceId: place_id,
+                      })
+                    }
+                    key={place_id}
+                  >
+                    {description}
+                  </li>
+                ))}
+              </>
+            </ul>
+            <button>Tour finden</button>
+          </SearchSuggestions>
+        )}
+      </LocationSearch>
+    </Wrapper>
+  )
+
+  function getGeolocationOfUser() {
```

Comment:
diese functions in eigene services verschieben

Reply:
Good point, thank you!